nrpy/infrastructures/CarpetX/general_relativity/BSSN_to_ADM.py

```diff
@@ -0,0 +1,164 @@
+"""
+Generates function to compute the ADM variables from the BSSN variables.
+
+Authors: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+         Samuel Cupp
+"""
+
+from typing import Union, cast
+from inspect import currentframe as cfr
+from types import FrameType as FT
+
+import nrpy.c_codegen as ccg
+import nrpy.c_function as cfc
+import nrpy.grid as gri
+import nrpy.helpers.parallel_codegen as pcg
+
+import nrpy.infrastructures.CarpetX.simple_loop as lp
+from nrpy.equations.general_relativity.BSSN_to_ADM import BSSN_to_ADM
+
+standard_ET_includes = [
```

Comment:
Centralize this; it's duplicated across multiple Python functions.

Reply:
Centralized in `CarpetX_include_header.py`